moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -393,7 +393,8 @@ void MotionPlanningFrame::changePlanningGroupHelper()
       // /std::shared_ptr<tf2_ros::Buffer> tf_buffer = context_->getFrameManager()->getTF2BufferPtr();
       std::shared_ptr<tf2_ros::Buffer> tf_buffer = moveit::planning_interface::getSharedTF();
 #endif
-      move_group_.reset(new moveit::planning_interface::MoveGroupInterface(node_, opt, tf_buffer, rclcpp::Duration(30)));
+      move_group_.reset(new moveit::planning_interface::MoveGroupInterface(node_, opt, tf_buffer,
+                                                                           rclcpp::Duration(std::chrono::seconds(30))));
```

Comment:
Assuming this is a shared pointer I'd change this to:

```suggestion
      move_group_ = std::make_shared<moveit::planning_interface::MoveGroupInterface>(node_, opt, tf_buffer,
                                                                           rclcpp::Duration::from_seconds(30));
```

Reply:
I've fixed this one but there are a ton of (around ~100) `new`s used to reset shared pointers in MoveIt 2. I am gonna go ahead and open an issue to track this and send a fix soon-ish.